app-android/src/main/java/com/ustadmobile/port/android/view/CourseDiscussionDetail2Fragment.kt

```diff
@@ -0,0 +1,355 @@
+package com.ustadmobile.port.android.view
```

Comment:
For the Jetpack compose UI, we are putting that inside the existing fragments. Please move the jetpack compose code to the existing fragment.